projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.ts

```diff
@@ -25,7 +25,6 @@ export class ProductFacetNavigationComponent implements OnInit, OnDestroy {
 
   activeFacetValueCode: string;
   searchResult: ProductSearchPage;
-  minPerFacet = 6;
```

Comment:
As this property is removed, the customers might get an error If they extended this component.
Should we create a deprecation ticket?

Reply:
We're going to have a big fat deprecation for the facet component anyway, we will do this later.